.github/workflows/cd.yml

```diff
@@ -53,6 +56,14 @@ jobs:
         aws s3 cp artifacts/*.rpm $s3_path/rpms/opendistro-security/
         aws cloudfront create-invalidation --distribution-id ${{ secrets.DISTRIBUTION_ID }} --paths '/downloads/*'
 
+    - name: Upload Artifacts to Maven Central
+      env:
+        SONATYPE_USER: ${{ secrets.SONATYPE_USER }}
+        SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+      run: |
+        cat <(echo -e "${{ secrets.PGP_PRIVATE_KEY }}") | gpg --batch --import
+        mvn -B clean deploy -Padvanced -Prelease -DskipTests -Dgpg.passphrase=${{ secrets.GPG_PASSPHRASE }}
```

Comment:
GPG_PASSPHRASE -> PGP_PASSPHRASE or PGP_PRIVATE_KEY -> GPG_PRIVATE_KEY

Reply:
@gaiksaya Could you confirm this? Looks like we have GPG_PASSPHRASE and PGP_PRIVATE_KEY

Reply:
Yes that's right @sujithvm along with pgp public key

Reply:
It should be consistent, either PGP or GPG. Public key is not a secret.

Reply:
Are you saying the key and passphrase itself or the names here must be consistent? I don't know how both were generated. 

I think we just private key to sign the artifact and public key seems to be already distributed here http://pool.sks-keyservers.net/pks/lookup?search=OpenDistroForElasticsearch&fingerprint=on&op=index

Reply:
Actually both, passphrase should be for the generated private key and names needs to be consistent as they refer to a related settings.

Public key does not need to be added to the secrets.

Reply:
Not referencing public key here so it is not added to secrets. 

Looking at this https://blog.sonatype.com/2010/01/how-to-generate-pgp-signatures-with-maven/ it looks fine to have pgp signing key with gpg passphrase and names need not be consistent.

The passphrase seems to be generated for the private key only.

@vrozov Am I missing something?

Reply:
> Not referencing public key here so it is not added to secrets.

Correct, I replied to @gaiksaya in response to 
> Yes that's right @sujithvm along with pgp public key

> Looking at this https://blog.sonatype.com/2010/01/how-to-generate-pgp-signatures-with-maven/ it looks fine to have pgp signing key with gpg passphrase and names need not be consistent.

There is only (pgp/gpg) signing key that is protected by (pgp/gpg) password. If you refer to it as a pgp key, should not it be pgp password/passphrase? Note that gpg is one of implementations of pgp spec, so it will be better either to use implementation (gpg) or spec (pgp) to refer to a (private) key and it's passphrase.

Reply:
Are you asking to rename the secret `secrets.GPG_PASSPHRASE` to `secrets.PGP_PASSPHRASE ` ? And have `-Dgpg.passphrase=${{ secrets.PGP_PASSPHRASE }}` ?

Reply:
Yes :)

Reply:
😄 I was assuming there was some mistake in the way the signatures were made making them incompatible